Configuration.php

```diff
@@ -16,6 +16,8 @@ class Configuration
     const DEFAULT_NOTIFY_THRESHOLD = 250000;
     const KEY_NOTIFY_EMAILS = 'notify_queue_threshold_emails';
     const KEY_NOTIFY_THRESHOLD = 'notify_queue_threshold_single_queue';
+    const LOG_FAILED_TRACKING_REQUESTS = 'log_failed_tracking_request_body';
```

Comment:
should we create an FAQ on where and how to use this ?

Reply:
Yep. I updated the FAQ file that's part of this repo:
https://github.com/matomo-org/plugin-QueuedTracking/pull/214/commits/4ea3e8bbc11844ffb60bf6ceee8379bc2820d616

Reply:
Also should we add this setting in install ? Something like [this](https://github.com/matomo-org/plugin-QueuedTracking/blob/4.x-dev/Configuration.php#L33..L35)

Reply:
I didn't see any need to add it at install as customers won't use it unless they're debugging.